exchange/exchange.go

```diff
@@ -405,6 +414,8 @@ func applyCategoryMapping(ctx context.Context, requestExt openrtb_ext.ExtRequest
 						//TODO: add metrics
 						//if mapping required but no mapping file is found then discard the bid
 						bidsToRemove = append(bidsToRemove, bidInd)
+						tempReason := fmt.Sprintf("Category mapping file for primary ad server: '%s', publisher: '%s' not found", primaryAdServer, publisher)
```

Comment:
Can you just call it `reason` instead of `tempReason`?

Reply:
Updated to be `reason`